### PR TITLE
README fix

### PR DESCRIPTION
**Pull request by Tommytrg**
Shekah Readme was not updated and it also had some errors.

In this PR I have updated the README to be consistent with the current state of Sheikah

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,46 @@
...
 
-We are using [Electron Log](https://github.com/megahertz/electron-log/) library and we log to both, console and the file in the following location:
-
-* **on Linux**: `~/.config/Sheikah/log.log`
-* **on MacOS**: `~/Library/Logs/Sheikah/log.log`
-* **on Windows**: `%USERPROFILE%\AppData\Roaming\Sheikah\log.log`
+## Sheikah Development
 
-### Development
+This is a Web/Desktop application built with [Electron](https://electronjs.org/). When developing Sheikah, we support **Node versions: 8, 9 and 10**. It might work with another version but we do not guarantee it will do in the future.
```
Comment (lrubiorod) [inline README.md]: Update number of versions supported

**Comment by lrubiorod** [comment]
Launching electron (yarn electron:serve) in Ubuntu 14.04 fails
Command to solve it should be included in README:
> yarn global add electron-prebuilt

File: README.md [hunk 2]
```diff
@@ -131,14 +107,17 @@ To build the application run: `yarn electron:build`, the build files are written
 
 You can read the [contributing guide](https://github.com/witnet/sheikah/blob/master/.github/CONTRIBUTING.md)
 
-#### Travis (continuous integration)
```
Comment (tmpolaczyk) [inline README.md]: This works now?

**Comment by aesedepece** [comment]
@Tommytrg Dude, you committed ` .witnet-rust-testnet-6-wallet` :rofl:

**Comment by tmpolaczyk** [comment]
> @Tommytrg Dude, you committed ` .witnet-rust-testnet-6-wallet` rofl

Quick, let's save his private key

**Pull request merged**